content/release-notes/1.20.0.md

```diff
@@ -0,0 +1,45 @@
+---
+date: 2020-10-06
+linktitle: "1.20.0"
+title: "1.20.0"
+kubernetes: "1.17, 1.18, and 1.19"
+---
+
+{{<features>}}
+* Added ability to delete disconnected nodes from the cluster management page of the Admin Console on embedded clusters.
+* Added template function support in [Collect](https://troubleshoot.sh/docs/collect/), [Analyze](https://troubleshoot.sh/docs/analyze/), and [SupportBundle](https://troubleshoot.sh/docs/support-bundle/collecting/) specs.
+* Added [template functions](/reference/template-functions/license-context/) for Sequence, Cursor, ChannelName, VersionLabel, ReleaseNotes, and IsAirgap.
+* Added support for automated (unattended, headless) upgrading of applications installed into airgapped instances using [KOTS CLI](/kots-cli/upstream/).
+* Added support for automated (unattended, headless) installs of applications into an embedded cluster (both airgap and online) using [KOTS CLI](/kotsadm/installing/installing-embedded-cluster/#headless-installs/ ).
+* Private images used by SupportBundle and Preflight [collectors](https://troubleshoot.sh/docs/collect/collectors/) are now automatically rewritten so they can be pulled from inside the cluster and are included in airgap bundles.
+{{</features>}}
+
+{{<changes>}}
+* Added support for Kubernetes 1.19 and dropped support for Kubernetes 1.15.
```

Comment:
Is this right?  We support the current 3 versions, so didn't we drop support for 1.16?

Reply:
Doh